frontend/src/components/EmailBox/EmailBox.tsx

```diff
@@ -12,16 +12,18 @@ function EmailBox({ emails }: { emails: EmailInfo[] }) {
 	const isOverflow = useIsOverflow(emailBoxContainer);
 
 	return (
-		<div
+		<section
 			className="email-box"
 			ref={emailBoxContainer}
 			// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
 			tabIndex={isOverflow ? 0 : undefined}
```

Comment:
Similarly here we shouldn't need this anymore as individual emails are now focusable.